### DISCUSSION TITLE
Ability to use self-hosted Bitwarden server

### DISCUSSION THREAD

**Discussion by EmilHeilbo** [minbrowser/min#1920] [Ideas]
As the title says, it would be nice to be able to use a self-hosted Bitwarden server - this is already supported in the 'bw' CLI tool with the command 'bw config server <URL>', so it would be nice if this would be supported in Min as well :)

**Comment by mfuezesi** [comment]
That would be super-nice!

**Reply by dogadaev** [reply]
Agree!
